i18n/de.json

```diff
@@ -3084,6 +3084,10 @@
     "translation": "Ungültige maximale Anzahl an Benutzers pro Team in Teameinstellungen.  Muss eine positive Zahl sein."
   },
   {
+    "id": "model.config.is_valid.max_channels.app_error",
```

Comment:
Just putting it in `en.json` is enough, our translation server takes care of that